SPECS/kernel-azure/config_aarch64

```diff
@@ -8934,6 +8934,9 @@ CONFIG_TRUSTED_KEYS=m
 CONFIG_ENCRYPTED_KEYS=m
 # CONFIG_KEY_DH_OPERATIONS is not set
 CONFIG_SECURITY_DMESG_RESTRICT=y
+CONFIG_PROC_MEM_ALWAYS_FORCE=y
```

Comment:
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/4a55880fa90902939c48b7d9035dc1ea4ee3a71f

Setting as this as it keeps the "Traditional /proc/pid/mem behavior"